docker-app/qfieldcloud/core/models.py

```diff
@@ -1246,6 +1251,20 @@ def save(self, recompute_storage=False, *args, **kwargs):
             self.file_storage_bytes = storage.get_project_file_storage_in_bytes(self.id)
         super().save(*args, **kwargs)
 
+    @property
+    def keep_storage(self) -> int:
```

Comment:
```suggestion
    def storage_versions(self) -> int:
```

I think the keyword here is "versions".